### PR TITLE
net-im/signal-desktop-bin: add new version 5.11.0

### PR DESCRIPTION
**Pull request by robert7k**

### CODE REVIEW

**Review by ionenwks — COMMENTED** [review]
Maybe time for some small ebuild updates? Some deps seem to be wrong notably.

**Comment by thesamesam** [comment]
ping

File: net-im/signal-desktop-bin/signal-desktop-bin-5.10.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+	opt/Signal/resources/app.asar.unpacked/node_modules/ref-napi/prebuilds/linux-x64/node.napi.node
+	opt/Signal/resources/app.asar.unpacked/node_modules/ringrtc/build/linux/libringrtc.node
+	opt/Signal/resources/app.asar.unpacked/node_modules/ringrtc/build/linux/libringrtc-x64.node
+	opt/Signal/resources/app.asar.unpacked/node_modules/sharp/build/Release/sharp.node
+	opt/Signal/resources/app.asar.unpacked/node_modules/sharp/vendor/8.10.6/lib/libvips-cpp.so.42
+	opt/Signal/resources/app.asar.unpacked/node_modules/zkgroup/libzkgroup.so
+	opt/Signal/resources/app.asar.unpacked/node_modules/zkgroup/libzkgroup-x64.so
+	opt/Signal/resources/app.asar.unpacked/node_modules/zkgroup/node_modules/ref-napi/build/Release/binding.node"
+
+RESTRICT="splitdebug"
```
[the remark this answers is not in the document]
Comment (robert7k) [reply net-im/signal-desktop-bin/signal-desktop-bin-5.10.0.ebuild]: It's supposed to address this:
https://bugs.gentoo.org/792309

**Comment by robert7k** [comment]
@ionenwks thanks for your suggestions, I tried to apply them. I also updated this PR to the most recent version 5.11.0.

File: net-im/signal-desktop-bin/signal-desktop-bin-5.10.0.ebuild (hunk 1 above)
[the remark this answers is not in the document]
Comment (ionenwks) [reply net-im/signal-desktop-bin/signal-desktop-bin-5.10.0.ebuild]: I see, let's keep it to be safe then. It'd be good if the bug reference was mentioned in the ebuild though, some ebuilds added this for other reasons.

**Review by ionenwks — APPROVED** [review]

**Pull request closed without merging**